catalog/USGS/catalog.jsonnet

```diff
@@ -50,8 +50,9 @@ local self_url = base_url + base_filename;
     ee.link.child_collection(
       'USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_COVER', base_url),
     ee.link.child_collection('USGS_NLCD_RELEASES_2020_REL_NALCMS', base_url),
-    //TODO uncomment when ready to publish
-    //ee.link.child_collection('USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS', base_url),
+    ee.link.child_collection('USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS', base_url),
+    //TODO (simonf) uncomment when ready to publish
+    //ee.link.child_collection('USGS_NLCD_RELEASES_2021_REL_NLCD', base_url),
```

Comment:
added the TRENDS_YEAR into the Catalog (it should be in there); wondering where the Trends_year jsonnet went (there should be a TRENDS and a TRENDS YEAR for the RCMAP V5). Thanks! @dpencosk

Reply:
I think your PR has an older version of this file and needs to be synced; I see them at head.

Reply:
Sounds good, I just want to make sure TRENDS and TRENDS YEAR are in there properly. Thanks!

Reply:
I'd say just manually update your block to look like the current one and add your change, that way we don't accidentally "roll back" when this one's done.
```
   ee.link.child_collection('USGS_NLCD_RELEASES_2016_REL', base_url),
    ee.link.child_collection('USGS_NLCD_RELEASES_2019_REL_NLCD', base_url),
    ee.link.child_collection('USGS_NLCD_RELEASES_2019_REL_RCMAP_V4_COVER', base_url),
    ee.link.child_collection(
      'USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_COVER', base_url),
    ee.link.child_collection(
      'USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS', base_url),
    ee.link.child_collection(
      'USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS_YEAR', base_url),
    ee.link.child_collection('USGS_NLCD_RELEASES_2020_REL_NALCMS', base_url),
    //TODO (simonf) uncomment when ready to publish
    //ee.link.child_collection('USGS_NLCD_RELEASES_2021_REL_NLCD', base_url),
```

Reply:
updated, it should be okay now I believe